pennylane_lightning/core/src/simulators/lightning_gpu/StateVectorCudaMPI.hpp

```diff
@@ -418,6 +418,28 @@ class StateVectorCudaMPI final
         }
     }
 
+    /**
+     * @brief Apply a single gate to the state vector.
+     *
+     * @param opName Name of gate to apply.
+     * @param controlled_wires Control wires.
+     * @param wires Wires to apply gate to.
+     * @param inverse Indicates whether to use adjoint of gate.
+     * @param params Optional parameter list for parametric gates.
+     * @param params Optional std gate matrix if opName doesn't exist.
+     */
+    template <template <typename...> class complex_t>
```

Comment:
Can we use `ComplexT` defined in Line 99 in this source file?

Reply:
No because I want to target both the `ComplexT` and `CFP_t` overloads above.